go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/synthetic-monitoring-agent
 
-go 1.21
+go 1.22
```

Comment:
Should we put down `1.22.0` instead?  I have `1.22.2` locally and I find without the full release number, it keeps pestering me about there not being a toolchain for `1.22`

Reply:
really?

let me look at what the logs say...

Reply:
the logs for the build do not say anything because I force GOTOOLCHAIN to local.

Reply:
I'll look at this later.